api/api.go

```diff
@@ -857,6 +857,29 @@ func CloudBackupStatusTypeToSdkCloudBackupStatusType(
 	}
 }
 
+func SdkCloudBackupStatusTypeToCloudBackupStatusString(
+	t SdkCloudBackupStatusType,
+) string {
+	switch t {
+	case SdkCloudBackupStatusType_SdkCloudBackupStatusTypeNotStarted:
```

Comment:
I think this is a problem with the SDK types. This fixes the issue with the status for the REST API but the SDK status strings still are "SdkCloudBackupStatusTypeDone". Why do users care about what interface was being used in the status string? And the status is already stored in the Backup Status field, so it doesn't need to have redundant information. The status string should just be "Done"

Reply:
Stings cannot be stored or described in gRPC proto file, only enums that is why it is called SdkCloudBackupStatusTypeDone.

Reply:
That just means the variable names need to be fixed. Please don't merge PRs when changes are requested on them

Reply:
You can look at the CloudMigrate status and phase strings on how to do this correctly: https://github.com/libopenstorage/openstorage/blob/master/api/api.pb.go#L992

Reply:
And this needs to be fixed for all the other SDK strings too

Reply:
@disrani-px the method you have above works well. Please write an issue so that we can fix all the values in the SDK to use that model. In the mean time, release-6.0 needs to have the current model.

Reply:
Actually, I'll create it.

Reply:
Filed #962